mockgen/version.1.11.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2010 Google Inc.
```

Comment:
```suggestion
// Copyright 2019 Google LLC
```
The guidance in LLC now.